docs/source/mock-contract.rst

```diff
@@ -123,6 +123,55 @@ Mock contract will be used to mock exactly this call with values that are releva
     });
   });
 
+Mocking multiple calls
+----------------------
+
+Mock contract allows to queue multiple mock calls to the same function. This can only be done if the function is not pure or view. That's because the mock call queue is stored on the blockchain and we need to modify it.
```

Comment:
Write a test where we call view function during a transaction. For instance, reading how many tokens do we own, then somehow update this number, then read again. Change the state accordingly. Also update the docs.

Reply:
Seems like this is not possible, because calling view functions uses a STATICCALL opcode.